cookbook/wandb_logged_chain/wandb_prompts/party_invite_prompt.py

```diff
@@ -5,12 +5,11 @@
 class PartyInvitePrompt(WandbPrompt):
```

Comment:
here and elsewhere, update to remove `Prompt` from the names i.e. `PartyInvite(WandbPrompt)`